skin-resque-frontend/src/pages/LoginRegister/LoginRegister.js

```diff
@@ -0,0 +1,77 @@
+import { Formik, Field } from 'formik';
+import { Link } from 'react-router-dom';
+import Heading from '../../components/Heading/Heading';
+import Button from '../../components/Button/Button';
+import './LoginRegister.scss'
+
+function LoginRegister({ isLogin }) {
+
+	return (
+		<div className='page' style={{ backgroundImage: `url('${process.env.PUBLIC_URL}/images/login-register.svg')` }}>
+			<div className='login-register'>
+				<Heading>{isLogin ? 'Log in' : 'Sign up'}</Heading>
+				<div className='form-links'>
+					<Formik
+						initialValues={
+							isLogin ?
+								{
+									username: '',
+									password: ''
+								}
+								:
+								{
+									username: '',
+									password: '',
+									email: ''
+								}
+						}
+						validate={values => {
+							const errors = {};
+							if (!isLogin && !values.email) {
+								errors.email = 'Email is required';
+							} else if (
+								!isLogin &&
+								!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)
+							) {
+								errors.email = 'Invalid email address';
+							} else if (!values.username) {
+								errors.username = 'Username is required';
+							} else if (!values.password) {
+								errors.password = 'Password is required';
+							}
+							return errors;
+						}}
+						onSubmit={(values, { setSubmitting }) => {
+							setTimeout(() => {
+								alert(JSON.stringify(values, null, 2));
+								setSubmitting(false);
+							}, 400);
+						}}
+					>{(formProps) => (
+						<form>
+							<div className='field'>
+								<Heading size='small'>Username</Heading>
+								<Field type='text' name='username' />
+							</div>
+							<div className='field'>
+								<Heading size='small'>Password</Heading>
+								<Field type='password' name='password' />
```

Comment:
make the fields all the same length
how it looks rn:
![image](https://user-images.githubusercontent.com/83237280/209566177-24ccd1fa-d7a7-4519-be09-47b1e4ae4b20.png)

how it should look:
![image](https://user-images.githubusercontent.com/83237280/209566191-b8fa959d-56e9-426d-8a01-d5d430c5f69d.png)